rust/relay/src/main.rs

```diff
@@ -113,7 +111,7 @@ async fn main() -> Result<()> {
             channel.join(
                 "relay",
                 JoinMessage {
-                    stamp_secret: hex::encode(server.auth_secret()),
```

Comment:
Ugh damn. I am sorry for that bug. I had the same issue recently and didn't think to check this usage.